packages/toolkit/scripts/build/copyFiles.ts

```diff
@@ -69,6 +69,19 @@ const tasks: CopyTask[] = [
         destination: path.join('src', 'stepFunctions', 'asl', 'aslServer.js'),
     },
 
+    // Serverless Land
+    {
+        target: path.join(
+            '../../node_modules/aws-core-vscode',
+            'src',
+            'awsService',
+            'appBuilder',
```

Comment:
join() isn't needed here either. I see that you are following the existing pattern of this file, but this file really should be updated.